test/DynamoCoreTests/Models/DynamoModelEventArgsTest.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using Dynamo.Graph.Nodes;
+using Dynamo.Models;
+using Dynamo.Utilities;
+using NUnit.Framework;
+using DynCmd = Dynamo.Models.DynamoModel;
+
+namespace Dynamo.Tests.ModelsTest
+{
+    /// <summary>
+    /// This test class created with the purpose of testing several methods/properties inside the next classes:
+    /// ZoomEventArgs
+    /// TaskDialogEventArgs
+    /// EvaluationCompletedEventArgs
+    /// DynamoModelUpdateArgs
+    /// FunctionNamePromptEventArgs
+    /// PresetsNamePromptEventArgs
+    /// ViewOperationEventArgs
+    /// PointEventArgs
+    /// WorkspaceEventArgs
+    /// ModelEventArgs
+    /// </summary>
+    [TestFixture]
+    class DynamoModelEventArgsTest : DynamoModelTestBase
+    {
+        //This method is for creating a Code Block node in the current model
+        private CodeBlockNodeModel CreateCodeBlockNode()
+        {
+            var cbn = new CodeBlockNodeModel(CurrentDynamoModel.LibraryServices);
+            var command = new DynCmd.CreateNodeCommand(cbn, 0, 0, true, false);
+
+            CurrentDynamoModel.ExecuteCommand(command);
+
+            Assert.IsNotNull(cbn);
+            return cbn;
+        }
+
+        /// <summary>
+        /// This test method will execute the  next ZoomEventArgs methods/properties:
+        ///  internal ZoomEventArgs(Point2D point)
+        ///  internal ZoomEventArgs(double zoom, Point2D point)
+        ///  internal Point2D Point { get; set; }
+        ///   internal bool hasPoint()
+        /// </summary>
+        [Test]
+        [Category("UnitTests")]
+        public void TestZoomEventArgs()
+        {
+            //Arrange/Act
+            //This will execute two constructor of the ZoomEventArgs class
+            var zoom = new ZoomEventArgs(new Point2D(100, 50));
+            var zoom2 = new ZoomEventArgs(100, new Point2D(200, 100));
+
+            //Assert
+            //Checking that the values for Point were stored correctly and also this will execute the Point.Get method
+            Assert.AreEqual(zoom.Point.X, 100);
+            Assert.AreEqual(zoom.Point.Y, 50);
+
+            Assert.AreEqual(zoom2.Point.X, 200);
+            Assert.AreEqual(zoom2.Point.Y, 100);
+
+            //This will execute the bool hasPoint() method, it returns true because we passed the Point as parameter
+            Assert.IsTrue(zoom.hasPoint());
+        }
+
+        [Test]
+        [Category("UnitTests")]
```

Comment:
Would you add some comments on this test?

Reply:
@QilongTang I added comments for this test case in the https://github.com/DynamoDS/Dynamo/pull/10612/commits/fd579961c315ade97b04f3634e7054724dc1ea9d commit.

Thanks